GiniVision/Assets/de.lproj/Localizable.strings

```diff
@@ -120,3 +120,49 @@
 "ginivision.images.openWithTutorialStep1" = "openWithTutorialStep1";
 "ginivision.images.openWithTutorialStep2" = "openWithTutorialStep2";
 "ginivision.images.openWithTutorialStep3" = "openWithTutorialStep3";
+
+"ginivision.digitalinvoice.screentitle" = "Digital Invoice";
+"ginivision.digitalinvoice.paybuttontitle.noinvoice" = "Pay";
+"ginivision.digitalinvoice.paybuttontitle" = "Pay %d/%d";
+"ginivision.digitalinvoice.paybuttontitle.accessibilitylabel" = "Pay %d out of %d";
+
+"ginivision.digitalinvoice.items" = "Items %d/%d";
+"ginivision.digitalinvoice.items.accessibilitylabel" = "Items %d out of %d";
+"ginivision.digitalinvoice.whatisthisbutton" = "What is this?";
+
+"ginivision.digitalinvoice.whatisthisactionsheet.title" = "Is it helpful?";
+"ginivision.digitalinvoice.whatisthisactionsheet.message" = "We extracted the detailed items from your invoice to help you calculate the amount to pay in case you want to return some goods or simply not fully-paying it.";
+"ginivision.digitalinvoice.whatisthisactionsheet.action.helpful" = "Yes, that's helpful";
+"ginivision.digitalinvoice.whatisthisactionsheet.action.nothelpful" = "No, I don't want to see it again";
```

Comment:
I'm confused by the button text here. They're just on an informational popup? The user won't dismiss that forever?

Or is the 'I don't want to see it again' meant to be about not wanting to see the entire digital invoice page again?

Reply:
It's the latter.